src/includes/head.pug

```diff
@@ -6,20 +6,16 @@ head
   link(rel='shortcut icon' href='assets/images/favicon.png' type='image/png')
   link(rel='icon' href='assets/images/favicon.png' type='image/png')
   link(rel='apple-touch-icon' href='assets/images/apple-touch.png')
-  link(rel='apple-touch-icon' sizes='72x72' href='assets/images/apple-touch-72.png')
-  link(rel='apple-touch-icon' sizes='114x114' href='assets/images/apple-touch-114.png')
-  link(rel='apple-touch-icon' sizes='144x144' href='assets/images/apple-touch-144.png')
-  
+
   link(rel='author' href='humans.txt')
   link(rel='stylesheet' type='text/css' href='bundle.css')
 
   meta(property='og:locale' content='en_GB')
   meta(property='og:type' content='article')
   meta(property='og:title' content='buildit@wiprodigital')
   meta(property='og:description' content='The future belongs to those who make')
-  meta(property='og:url' content='https://buildit.digital/')
-  meta(property='og:site_name' content='buildit_')
-  meta(property='og:image' content='https://buildit.digital/assets/images/buildit-logo-white.png')
-  meta(property='article:published_time' content='Fri March 17, 2017')
+  meta(property='og:url' content='https://buildit.wiprodigital.com/')
```

Comment:
this sheet needs to be parametrised!!!

Reply:
It _should_ be, yeah. I don't think it's worth trying to fix that at this point though - let's just make sure the new site gets it right :-)